examples/flows/standard/basic-code-first/quickstart.ipynb

```diff
@@ -0,0 +1,323 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Getting started with prompt flow code-first experience\n",
+    "\n",
+    "**Learning Objectives** - Upon completing this tutorial, you should be able to:\n",
+    "\n",
+    "- Write LLM application using notebook and visualize the trace of your application.\n",
+    "- Convert the application into a flow and batch run against multi lines of data.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 0. Install dependent packages"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%pip install -r ../../requirements.txt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Trace your application with promptflow\n",
+    "\n",
+    "Assume we already have a python function that calls OpenAI API. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with open(\"llm.py\") as fin:\n",
+    "    print(fin.read())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note: before running below cell, please configure required environment variable `AZURE_OPENAI_API_KEY`, `AZURE_OPENAI_API_BASE` by create an `.env` file. Please refer to [.env.example](.env.example) as an template."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# control the AOAI deployment (model) used in this example\n",
+    "deployment_name=\"text-davinci-003\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from llm import my_llm_tool\n",
+    "# pls configure `AZURE_OPENAI_API_KEY`, `AZURE_OPENAI_API_BASE` environment variables first\n",
```

Comment:
how about make something like `os.envrion`? otherwise it's very easy to ignore/skip this (for example, me)

Reply:
I expect user to set the .env file here actually, any good idea to guide user here?

Reply:
I just read the notebook in detail, which looks good enough to me.